src/controllers/verifyOTP.js

```diff
@@ -19,17 +25,34 @@ const verifyOTP=async(req,res)=>{
         },
     })
     if(!user){
-        return res.status(200).send("invalid OTP!")
+        return res.status(200).json({message:"invalid OTP!"})
     }
     //clear the otp and log the user in
     else{
```

Comment:
Avoid elses as much as you can instead you can use return

Reply:
how can i return while i'm going to blacklist the token after successfull login i was putting the else in order to make sure javascript follow the flow of codes but i've removed the else